### ISSUE TITLE
Support for themes

### ISSUE THREAD

**Issue by shalvah** [knuckleswtf/pastel#3]
Identical to https://github.com/knuckleswtf/pastel-js/issues/1

Our templates are written in Blade, but other than that, it's the same requirement. If a new theme is added to one library, it can easily be ported to the other.

**Comment by tsprings** [comment]
I would suggest that allowing for custom styles should maybe be on the roadmap before templates.

**Comment by shalvah** [comment]
Hmm. I never really thought about custom styles, but yeah, you're right. That would make a lot of people happy, for little cost.

That seems a pretty straightforward implementation: add a `css` property to the front matter where you can specify files containing your own styles and Pastel turns them into `<link>` tags and inserts them after its CSS.

**Comment by ajcastro** [comment]
I dive into the codebase of pastel. I think this the part where we can customize it? 
![image](https://user-images.githubusercontent.com/4918318/95966498-c96ba100-0e3d-11eb-9f63-d6f2c7e176c7.png)

Maybe we can add a config file and put there a list of custom css styles paths
`config/pastel.php`
```php
[
  'css' => [
    get_css_link_tag('highlight-darcula'),
    'path-to-my-custom2.css',
  ]
]
```
This way we can disable the `highlight-darcula`, just my idea/suggestion.

Related: https://github.com/knuckleswtf/scribe/issues/113

**Comment by seanhudson** [comment]
Is there any progress on CSS styling?

**Comment by shalvah** [comment]
No, it's not been a priority. I'm working on Scribe v3, which will likely include that support directly.
